### ISSUE TITLE
Invalidating caches for news extension pages

### ISSUE THREAD

**Issue by ax-ha** [toumoro/tm_cloudfront#9]
Are there any known problems when the extension is running in combination with the news extension?
Currently my cdn cache for the news detail page is not cleared when a news entry is edited :-(
(Typo3 Version 11)
I would be very happy to receive a helpful answer.

**Comment by simouel** [comment]
I don't think we support individual news invalidation, but you can specify the news page id in tsconfig. All news cache will be cleared.
```
TCEMAIN {   
    clearCacheCmd = uid of the news page
}
```

**Comment by ax-ha** [comment]
Hello simouel,
thanks a lot for your reply. Sorry for the long delay. I had some days off.
Unfortunatly, this solution is flushing all news pages which is not that good when you have hundreds of them.
I will have to find a solution for this in the future. Perhaps a deeper look into the hooks of the news extension might be a good idea :-)
